fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentMapper.java

```diff
@@ -67,6 +67,9 @@ public FluentClient map(CodeModel codeModel, Client client) {
             fluentClient.getExamples().addAll(examples);
         }
 
+        // live tests
+        fluentClient.getLiveTests().addAll(client.getLiveTests().stream().map(liveTests -> FluentLiveTestsMapper.getInstance().map(liveTests, fluentClient, codeModel, fluentJavaSettings)).collect(Collectors.toList()));
```

Comment:
nit, use some line break to avoid too long of the code line.

Reply:
Changed.